web/src/settings_invites.ts

```diff
@@ -103,13 +121,20 @@ function populate_invites(invites_data: {invites: Invite[]}): void {
                 item.expiry_date_absolute_time = timerender.absolute_time(item.expiry_date * 1000);
             }
             item.is_admin = current_user.is_admin;
-            item.disable_buttons =
+            item.disable_revoke_button =
                 item.invited_as === settings_config.user_role_values.owner.code &&
                 !current_user.is_owner;
+            item.disable_edit_button =
```

Comment:
I think permission to create permission should also be checked here itself. We can avoiding setting `disable_edit_button`  value twice.